src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -261,6 +263,56 @@ private static void AddRequiresLicenseAcceptance(Document document, IDictionary<
             }
         }
 
+        private static void AddPackageTypes(Document document, IDictionary<string, string> package)
+        {
+            string value;
```

Comment:
done.

---
In reply to: [75554919](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75554919) [](ancestors = 75554919)